### PR TITLE
Add options for Grinder-systemtest job

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Resolves https://github.com/AdoptOpenJDK/openjdk-tests/issues/891
Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Testing run of an openjdk-systemtest using developer branch from AdoptOpenJDK/systemtest AdoptOpenJDK/stf and eclipse/openj9-systemtest repo: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder-systemtest/14/

Testing run of an openj9-system test using developer branch from AdoptOpenJDK/systemtest AdoptOpenJDK/stf and eclipse/openj9-systemtest repo: 
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder-systemtest/15/console

Testing using default values of repos: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder-systemtest/18/

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -193,6 +193,37 @@ def setup() {
 			OPENJ9_BRANCH_OPTION = (params.OPENJ9_BRANCH) ? "--openj9_branch ${params.OPENJ9_BRANCH}" : ""
 			OPENJ9_SHA_OPTION = (params.OPENJ9_SHA) ? "--openj9_sha ${params.OPENJ9_SHA}" : ""
 
+			// system test repository exports to be used by systemtest/common.xml
+			if (params.ADOPTOPENJDK_SYSTEMTEST_REPO) {
+				env.OPENJDK_SYSTEMTEST_REPO = "${params.ADOPTOPENJDK_SYSTEMTEST_REPO}"
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase]: - Could we have the environment variable `OPENJDK_SYSTEMTEST_REPO ` match with the parameter `ADOPTOPENJDK_SYSTEMTEST_REPO`? Having a mismatching name could confuse the users.
- We should be able to have `params.ADOPTOPENJDK_SYSTEMTEST_REPO` directly. There is no need to use `"${ ...}"`

File: buildenv/jenkins/JenkinsfileBase [hunk 2]
```diff
@@ -193,6 +193,37 @@ def setup() {
...
 
+			// system test repository exports to be used by systemtest/common.xml
+			if (params.ADOPTOPENJDK_SYSTEMTEST_REPO) {
+				env.OPENJDK_SYSTEMTEST_REPO = "${params.ADOPTOPENJDK_SYSTEMTEST_REPO}" 
+			}
+			
+			if (params.ADOPTOPENJDK_SYSTEMTEST_BRANCH) {
+				env.OPENJDK_SYSTEMTEST_BRANCH = "${params.ADOPTOPENJDK_SYSTEMTEST_BRANCH}"
+			} else {
+				echo "ADOPTOPENJDK_SYSTEMTEST_BRANCH not set"
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase]: This is an optional parameter and it is only used for system test. I would prefer not to echo it in the base. If msg is needed, please do it in systemtest common.xml. Same for the rest echo msg.

File: buildenv/jenkins/JenkinsfileBase (hunk 1 above)
Comment (Mesbah-Alam) [reply buildenv/jenkins/JenkinsfileBase]: Changes made! Thanks for the suggestions.

File: buildenv/jenkins/JenkinsfileBase (hunk 2 above)
Comment (Mesbah-Alam) [reply buildenv/jenkins/JenkinsfileBase]: Agreed, removed those else{..} blocks.

**Comment by llxia** [comment]
In Grinder-systemtest, I think we should remove `OPENJ9_REPO` and `OPENJ9_BRANCH` or move them at the bottom. It is unlikely for users to change both system tests and TKG.

**Comment by Mesbah-Alam** [comment]
@llxia - makes sense, will move those TKG related parameters down in Grinder-systemtest.

**Pull request merged**